src/data/critical-constants-acentric-factor.ts

```diff
@@ -15,4 +15,268 @@ export const DIPPRCriticalConstantsAndAcentricFactorDictionary: Record<
     criticalCompressibilityFactor: 0.221,
     acentricFactor: 0.262493,
   },
+  Acetamide: {
+    name: 'Acetamide',
+    formula: 'C2H5NO',
+    CAS: '60-35-5',
+    molecularWeight: 59.0672,
+    criticalTemperature: 761,
+    criticalPressure: 6.6,
+    criticalVolume: 0.215,
+    criticalCompressibilityFactor: 0.224,
+    acentricFactor: 0.421044,
+  },
+  AceticAcid: {
+    name: 'Acetic acid',
+    formula: 'C2H4O2',
+    CAS: '64-19-7',
+    molecularWeight: 60.052,
+    criticalTemperature: 591.95,
+    criticalPressure: 5.786,
+    criticalVolume: 0.177,
+    criticalCompressibilityFactor: 0.208,
+    acentricFactor: 0.466521,
+  },
```

Comment:
```suggestion
  },
  
```
@saikat-samanta  let's add a new line after  compound object data is finished